dev/ci/ci-fourcolor.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# $0 is not the safest way, but...
```

Comment:
what's this comment for?

Reply:
I've no idea where it comes from

Reply:
it comes from ci-mathcomp.sh... I'll let you dig